pkgs/development/libraries/comedilib/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, flex
+, yacc
+, xmlto
+, docbook_xsl
+, docbook_xml_dtd_44
+, swig
+, perl
+, python3
+# Can be used to create pdf documentation, but it makes the build
+# fail and it's not worth the closure size increase
+# , dblatex
```

Comment:
Can you remove these 3 lines? That `dblatex` can't be found is pretty visible in configure phase, if one ever decides to try to re-add this.

Reply:
:+1:.

Reply:
:1234: .